portfolio.css

```diff
@@ -0,0 +1,574 @@
+ }
+
+.contact-cta-button {
+  display: flex;
+  width: 181px;
+  height: 61px;
+  padding: 16px 32px;
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 10px;
+  background: linear-gradient(180deg, rgba(240, 239, 239, 0.91) 48.96%, rgba(240, 239, 239, 0.36) 99.99%, rgba(240, 239, 239, 0.00) 100%);
+  box-shadow: 0px 8px 32px 0px rgba(255, 255, 255, 0.25);
+  text-align: center;
+  color: var(--portfolio-black-text, #292929);
+  font-family: "SF Pro Text";
+  font-size: 22px;
+  font-weight: 600;
+  line-height: 32px;
+  text-decoration: none;
+  cursor: pointer;
+  margin-bottom: 0;
+}
+.contact-cta-button:hover {
+  opacity: 0.6;
+}
+
+.parent-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh; /* Adjust this as needed for vertical centering */
+  background: #313131; /* Set a background color */
+}
+
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  background: #1D1C1C;
+}
+
+.footer-text {
+  color: var(--portfolio-text, #F5F1F1);
+  font-family: "SF Pro Text";
+  font-size: 25px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 32px; /* 128% */
+  text-align: left;
+}
+
+.linkedin-text {
+  color: var(--portfolio-text, #F5F1F1);
+  font-family: "SF Pro Text";
+  font-size: 25px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: left;
+  text-decoration-line: underline;
+}
+
+
+
+@media screen and (max-width: 480px) {
+ 
+   .nav-bar {
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    padding: 10px 20px;
+  }
+
+  .nav-div {
+    text-align: center;
+    margin-top: 10px;
+  }
+
+  .nav-list {
+    padding: 10px;
+    border-radius: 0;
+    flex-direction: column;
+  }
+  .list-item {
+    /* ... Other styles ... */
+    padding: 10px 0; /* Adjust padding */
+  }
+
+  .text-image {
+    /* ... Other styles ... */
+    gap: 20px; /* Decrease the gap */
+    padding: 40px 20px; /* Adjust padding */
+  }
+
+  .content {
+    /* ... Other styles ... */
+    gap: 15px; /* Decrease the gap */
+  }
+
+  /* Adjust font sizes for headings and paragraphs */
+  h2 {
+    font-size: 24px;
+  }
+
+  .designer {
+    font-size: 18px;
+  }
+
+
+  #text {
+    font-size: 32px;
+    line-height: 40px;
+  }
+  .card-container {
+    /* ... Other styles ... */
+    gap: 15px; /* Adjust the gap */
+  }
+
+  /* Adjust font sizes for headings and descriptions */
+  .skills-heading,
+  .projects-heading {
+    font-size: 28px;
+  }
+
+  .card-description,
+  .project-subheading {
+    font-size: 14px;
+  }
+
+
+  .contact-container {
+    padding: 20px;
+  }
+
+  .contact-heading {
+    font-size: 28px;
+  }
+
+  .contact-text,
+  #contact {
+    font-size: 18px;
+  }
+
+  .contact-cta-button {
+    margin-top: 15px;
+  }
+
+  .footer {
+    padding: 10px;
+  }
+
+  .footer-text,
+  .linkedin-text {
+    font-size: 18px;
+  }
+}

```

Comment:
There is a lot of CSS for a page that is relatively simple. 
That is more CSS than sometimes a multi-page app could have. 
This is all natural - it always happens in the beginning when first learning CSS. 
Try to see if you can refactor the CSS:
- maybe the same is repeated some places
- maybe the same element is styled in different ways and only one of the ways takes effect
- maybe there are styling properties that just do nothing, do not affect the design at all

Analyse the CSS as a detective and see if you can optimise at least a little bit 🕵🏽